app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,67 @@
+import { Prism } from '@mantine/prism';
+import { Center, Divider, Grid, Paper, Space, Tabs, Text } from '@mantine/core';
+import { ArtifactResourceType } from '@/util/types/fhir';
+import { useRouter } from 'next/router';
+import { trpc } from '@/util/trpc';
+
+/**
+ * Component which renders a page that displays the JSON data of a resource. Also will eventually
+ *  provide the user with the ability to make review comments and visualize previous review comments.
+ */
+export default function CommentPage() {
+  const router = useRouter();
+  const { resourceType: resourceType, id: resourceID, authoring } = router.query;
+  let resource;
+
+  if (authoring === 'true') {
+    resource = trpc.draft.getDraftById.useQuery({
+      id: resourceID as string,
+      resourceType: resourceType as ArtifactResourceType
+    });
+  } else {
+    resource = trpc.service.getArtifactById.useQuery({
+      id: resourceID as string,
+      resourceType: resourceType as ArtifactResourceType
+    });
+  }
+
+  return (
+    <div>
+      <Center>
+        <Text c="gray" fz="xl">
+          {authoring === 'true'
+            ? `Reviewing Draft ${resourceType}/${resourceID}`
+            : `Reviewing ${resourceType}/${resourceID}`}
+        </Text>
+      </Center>
+      <Divider my="md" mt={14} />
+      <Grid>
+        <Grid.Col span={6}>
+          <Tabs variant="outline" defaultValue="addComments">
+            <Tabs.List>
+              <Tabs.Tab value="addComments">Add comment</Tabs.Tab>
+              <Tabs.Tab value="viewComments">View Comments</Tabs.Tab>
+            </Tabs.List>
+            <Tabs.Panel value="addComments" pt="xs">
+              Components to add review comment goes here
+            </Tabs.Panel>
+            <Tabs.Panel value="viewComments" pt="xs">
+              Components to view comments go here
+            </Tabs.Panel>
+          </Tabs>
+        </Grid.Col>
+        <Grid.Col span={6}>
+          <Space />
+          <Text c="gray" fz="sm">
+            Current JSON Content
+          </Text>
+          <Paper withBorder>
+            <Prism language="json" colorScheme="light" styles={{ scrollArea: { height: 'calc(100vh - 150px)' } }}>
+              {resource.data ? JSON.stringify(resource.data, null, 2) : ''}
+            </Prism>
+          </Paper>
+        </Grid.Col>
```

Comment:
This doesn't require a change in this PR, but once we have the UI in place for adding comments, it might be nice to extend the width of the JSON content so that the user does not need to scroll as much when viewing the JSON. But this will depend on how the comments look once that functionality is in place :) just something to keep in mind for now